inbm/dispatcher-agent/dispatcher/source/linux_gpg_key.py

```diff
@@ -61,4 +62,4 @@ def add_gpg_key(remote_key_path: str, key_store_name: str) -> None:
         raise SourceError(f"Error getting GPG key from remote source: {e}")
 
     except subprocess.CalledProcessError as e:
-        raise SourceError(f"Error running gpg command to dearmor key: {e}")
+        raise SourceError(f"Error running GPG command to dearmor key: {e}")
```

Comment:
Use a more descriptive exception message that includes the `key_store_name` when the GPG command fails. [medium]

Reply:
in this case, I don't think that is necessary